website/snippets/_sso-docs-mt-available.md

```diff
@@ -0,0 +1,7 @@
+:::info Enterprise feature
+
+This guide describes a feature of the dbt Cloud Enterprise plan. If you’re interested in learning more about an Enterprise plan, contact us at <sales@getdbt.com>.
+
+This SSO configuration document applies to multi-tenant Enterprise deployments only. [Single-tenant](/docs/cloud/about-cloud/tenancy#single-tenant) Virtual Private users can [email dbt Cloud Support](mailto:support@getdbt.com) to set up or update their SSO configuration. 
```

Comment:
@ajaythomas @BrJan i created a snippet since this is repeated in multiple places. also suggested to reword it based on style guide. let me know what you think